R/options.R

```diff
@@ -4,20 +4,4 @@
 #' @description
 #' See below for options available in the \{cards\} package
 #'
-#' @section cards.round_type:
-#' There are two types of rounding types in the \{cards\} package that are implemented
-#' in `label_round()`, `alias_as_fmt_fn()`, and `apply_fmt_fn()` functions' `round_type`
-#' argument.
```

Comment:
Update this section to say that these functions are affected by the option, and remove the part  about the argument (as it doesn't exist)